### PR TITLE
Make sure we have enough free space before snapshotting

### PR DESCRIPTION
**Pull request by dgutu-suse**
Based on the question from:
https://bugzilla.suse.com/show_bug.cgi?id=998360#c19

### CODE REVIEW

File: tests/console/snapper_cleanup.pm [hunk 1]
```diff
@@ -24,6 +24,11 @@ sub snapper_cleanup {
     my $free_space      = script_output("$btrfs_fs_usage | sed -n '7p' | awk -F ' ' '{print\$3}'");
     my $excl_free_space = ($fs_size / 2);
 
+    diag 'Check if we have enough free space to creat snapshots';
```
Comment (okurz) [inline tests/console/snapper_cleanup.pm]: s/creat/create/

File: tests/console/snapper_cleanup.pm [hunk 2]
```diff
@@ -24,6 +24,11 @@ sub snapper_cleanup {
...
+    if ($free_space < $excl_free_space) {
+        die("The test most probably will fail, FS has less fre space then expected");
```
Comment (okurz) [inline tests/console/snapper_cleanup.pm]: s/fre/free/ and I recommend to use "die" without parentheses and just single ticks

File: tests/console/snapper_cleanup.pm (hunk 1 above)
Comment (dgutu-suse) [inline tests/console/snapper_cleanup.pm]: Thx, done.

File: tests/console/snapper_cleanup.pm (hunk 2 above)
Comment (dgutu-suse) [inline tests/console/snapper_cleanup.pm]: Thx, done.

File: tests/console/snapper_cleanup.pm [hunk 3]
```diff
@@ -24,6 +24,11 @@ sub snapper_cleanup {
     my $free_space      = script_output("$btrfs_fs_usage | sed -n '7p' | awk -F ' ' '{print\$3}'");
     my $excl_free_space = ($fs_size / 2);
 
+    diag 'Check if we have enough free space to create snapshots';
+    if ($free_space < $excl_free_space) {
+        die "The test most probably will fail, FS has less free space then expected";
```
Comment (okurz) [inline tests/console/snapper_cleanup.pm]: * single ticks

and better write 'The filesystem has leess free space than this tests expects, the test will most likely fail'

Comment (dgutu-suse) [inline tests/console/snapper_cleanup.pm]: Done

File: tests/console/snapper_cleanup.pm [hunk 4]
```diff
@@ -61,6 +66,7 @@ sub run() {
         assert_script_run("$snapper get-config");    # get initial cfg
         assert_script_run("$snapper list");          # get initial list of snap's
         assert_script_run("$snapper set-config NUMBER_MIN_AGE=0");
+        assert_script_run("$snapper btrfs filesystem usage / --raw");    # get FS overview
```
Comment (DimStar77) [inline tests/console/snapper_cleanup.pm]: https://openqa.opensuse.org/tests/395051#step/snapper_cleanup/10

Unknown command 'btrfs'

**Pull request merged**